### DISCUSSION TITLE
Error drawer for content editors in Engage

### DISCUSSION THREAD

**Discussion by tesshannel** [twilio-labs/paste#3380] [Office hours] [answered]
### Background

I am designing a space to house error messages in Engage content editors (mobile and email). The first use case is to surface merge tag validation errors, but I expect it to eventually house other types of error messages as well.

The closest similar discussion I've found is [this](https://github.com/twilio-labs/paste/discussions/834) one, which proposes using a combo of alert + modal, but I'm worried that will create an uncomfortable editing experience for users.

### Feedback

- Can the alert component be made to collapse and expand as the designs show?
- Is there another approach you'd recommend for building a consolidated error space?

### Stage of design

Lo-mid fidelity

### Relevant links

https://www.figma.com/file/5TIyKoIRyuWaAth7RSyxjk/Content-Validation-%2B-Errors?type=design&node-id=1-1494&mode=design&t=8ISG12Cqpk50n70f-11

### Are you looking for visual design support?

- [ ] Yes

**Comment by SiTaggart** [comment, marked as answer]
Thanks for coming to hang in Office hours today! 

A recording of the meeting can be found here https://twilio.zoom.us/rec/share/F1_WAOJASrOxIynE3cpIjoqiJYt9CQX7tHPikqPF64M5UuLbsAyUsJlcyY-v6gtr.fsLh97Ectqi3dlx2 (The passcode can be found in the #help-design-system Slack channel)
